src/System.Binary.Base64/System/Binary/Base64Decoder.cs

```diff
@@ -294,9 +211,10 @@ public TransformationStatus Transform(ReadOnlySpan<byte> source, Span<byte> dest
         /// <returns>Number of bytes written to the buffer.</returns>
         public static int DecodeInPlace(Span<byte> buffer)
         {
+            // TODO: Fix
```

Comment:
It would be great if the TO described what the work is. I want to maximize the bus number ;-)